addons/website_blog/controllers/main.py

```diff
@@ -92,16 +92,19 @@ def _prepare_blog_values(self, blogs, blog=False, date_begin=False, date_end=Fal
             domain += [('tag_ids', 'in', active_tags.ids)]
 
         if request.env.user.has_group('website.group_website_designer'):
-            count_domain = domain + [("website_published", "=", True), ("post_date", "<=", fields.Datetime.now())]
+            count_domain = domain + [("website_published", "=", True)]
+            scheduled_domain = domain + [("scheduled", "=", True)]
             published_count = BlogPost.search_count(count_domain)
             unpublished_count = BlogPost.search_count(domain) - published_count
-
+            scheduled_count = BlogPost.search_count(scheduled_domain)
             if state == "published":
-                domain += [("website_published", "=", True), ("post_date", "<=", fields.Datetime.now())]
+                domain += [("website_published", "=", True)]
             elif state == "unpublished":
-                domain += ['|', ("website_published", "=", False), ("post_date", ">", fields.Datetime.now())]
+                domain += [("website_published", "=", False)]
+            elif state == "scheduled":
+                domain += [("scheduled", "=", True)]
```

Comment:
As discussed: should the scheduled posts be part of the unpublished posts? Wouldn't it be more interesting to clearly differentiate the 2 types? (I don't have an answer, I think both are valid. Just asking to be sure it does what we want.)

Reply:
A debate to have with a PO I guess. For me a scheduled page is by definition not published, so we have a union here. It worth a discussion about it. We'll see.